sparkling-water-mojo-scorer/README.md

```diff
@@ -0,0 +1,32 @@
+# Example of using MOJO2 from PySparkling
+Multi-threaded MOJO scorer using Sparkling Water.
+
+## Requirements
+
+  - Java 8
+  - Configure `SPARK_HOME` to point to Spark distribution directory
+  - Configure `SPARKLING_HOME` to point to to Sparkling Water distribution directory
+  - Configure `DRIVERLESS_AI_LICENSE_FILE` to point to an active DAI license.sig file
+
+## (optional) 
+
+  - Configure `MASTER` to point to the master url for the cluster
+  - Create and use a Conda environment
+
+    ```
+    conda create -n mojo python=3.6
```

Comment:
do we want to be dependent on recommending `conda`. If anything, I would suggest shipping a template dockerfile.